datalad_next/tests/test_create_sibling_webdav.py

```diff
@@ -368,3 +371,136 @@ def query(self, *args, **kwargs):
             credman=MockCredentialManager("u", "v"))
         eq_(result[0], None)
         eq_(result[1][0][1]["realm"], None)
+
+
+@with_credential(
+    'dltest-mywebdav', user=webdav_cred[1], secret=webdav_cred[2],
+    type='user_password')
+@with_tree(tree={'sub': {'f0': '0'},
+                 'sub2': {'subsub': {'f1': '1'},
+                          'f2': '2'},
+                 'f3': '3'})
+@with_tempfile
+@serve_path_via_webdav(auth=webdav_cred[1:])
+def test_existing_switch(localpath, remotepath, url):
```

Comment:
Here is an interesting twist to this test that covers a conflict scenario that will occur often:

When a webdav sibling is in annex-mode, the annex object tree will be in the root. This means that for recursively deposited subdatasets, they is a built-in conflict (potential) for any  subdataset with a three-character hex-compliant mountpoint. When a git-sibling is also included, there will be a hard conflict with `3f7` and `eb3`. These should be handled gracefully somehow -- worth including here.

Reply:
Done.

At least superficially. The real test would be a subsequent `push`